### PR TITLE
[charts/portal] MySQL InPlace upgrade from 5.7 to 8.0.26

### PR DESCRIPTION
**Pull request by kk632390**
**Description of the change**
Added documentation for upgrading MySQL 5.7 to 8.0.26

**Benefits**
Portal supported OnPremise MySQL Database upgrade steps are documented 

**Drawbacks**
None

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: charts/portal/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This Chart deploys the Layer7 API Developer Portal on a Kubernetes Cluster using
 Solutions & Patches](https://techdocs.broadcom.com/us/product-content/recommended-reading/technical-document-index/ca-api-developer-portal-solutions-and-patches.html)
 
 ### Production
-- A dedicated MySQL 5.7/8.0.22 server [TechDocs](https://techdocs.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-0-2/install-configure-and-upgrade/install-portal-on-docker-swarm/configure-an-external-database.html#concept.dita_18bc57ed503d5d7b08bde9b6e90147aef9a864c4_ProvideMySQLSettings)
+- A dedicated MySQL 8.0.22/8.0.26 server [TechDocs](https://techdocs-author.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-1/install-configure-and-upgrade/install-portal-on-docker-swarm/configure-an-external-database.html)
```
Comment (SatishKoney-BRCM) [inline charts/portal/README.md]: let's keep tech-docs...just make sure that the link works..

File: charts/portal/README.md [hunk 2]
```diff
@@ -59,6 +59,43 @@ To delete API Portal installation
 
 *Additional resources such as PVCs and Secrets will need to be cleaned up manually. This protects your data in the event of an accidental deletion.* 
 
+## Upgrade External Portal Database to MySQL 8.0
+MySQL 8.0 is supported as external database starting from Portal 5.0 CR 1. This section helps you understand the overall process of upgrading an existing Portal database running MySQL 5.7 to MySQL 8.0.
+
+Before starting upgrade of Database follow the sections **Before You Begin** and **Check Database Compatibility** from (https://techdocs-author.broadcom.com/us/en/ca-enterprise-software/layer7-api-management/api-developer-portal/5-1/install-configure-and-upgrade/install-portal-on-docker-swarm/upgrade-portal-database-to-mysql-8.html)
```
Comment (SatishKoney-BRCM) [inline charts/portal/README.md]: same here

**Review by SatishKoney-BRCM — CHANGES_REQUESTED** [review]

File: charts/portal/README.md (hunk 2 above)
Comment (kk632390) [reply charts/portal/README.md]: updated

File: charts/portal/README.md (hunk 1 above)
Comment (kk632390) [reply charts/portal/README.md]: updated

**Review by ksaladi — APPROVED** [review]
LGTM

**Review by ksaladi — APPROVED** [review]
LGTM

**Review by ksaladi — APPROVED** [review]
LGTM

**Pull request merged**